Domain/Inventory_Class/Inventory.cs

```diff
@@ -0,0 +1,114 @@
+﻿using InventoryManagementSystem.Domain.General;
+using InventoryManagementSystem.Domain.Product_Class;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace InventoryManagementSystem.Domain.Inventory_Class
+{
+    public class Inventory
+    {
+        private List<Product> products = new List<Product>();
+
+        public void AddProduct(string name, Price price, int quantity)
+        {
+            products.Add(new Product(name, price, quantity));
+            Console.WriteLine("Added successfully.\r\nPress enter key to Back.");
+        }
+
+        public void ViewProducts()
+        {
+            if (products.Count == 0)
+            {
+                Console.WriteLine("No products in the inventory.");
+                return;
+            }
+
+            Console.WriteLine("\nList of Products:\n********************************");
+            foreach (var product in products)
+            {
+                Console.WriteLine($"Name: {product.Name}\nPrice: {product.Price:C}\nQuantity: {product.Quantity}" +
+                    $"\n********************************");
+            }
+            Console.WriteLine("Press enter key to Back. \r\n");
+            Console.ReadKey();
+        }
+
+        public void EditProduct(string name)
+        {
+            var product = products.Find(p => p.Name.Equals(name, StringComparison.OrdinalIgnoreCase));
+            if (product == null)
+            {
+                Console.WriteLine("Product not found!");
+                Console.WriteLine("Press enter key to Back. \r\n");
```

Comment:
the methods in `Inventory` class should just do direct CRUD and update products lists. If we want the user to input anything or show a message to the user for any info, this should be out of the inventory class, should be the responsibility of the caller. i.e. , `EditProduct` should receive all info that need updated and the method should update the product directly.

Reply:
lets do this change to all methods in this class